packages/multiselect/src/react/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { Option } from './types'
+
+export function arraysAreEqual<T>(a: T[] = [], b: T[] = []) {
+  return a.length === b.length && a.every(item => b.includes(item))
```

Comment:
Array.includes doesn't work in IE11.  Do we ship a polyfill?  If not, might just want to use indexOf as is done elsewhere in the PR.